### PR TITLE
Various security center updates

### PR DESCRIPTION
**Pull request by danh-arm**
* Refer to Linaro security policy for infrastructure issues
* Mention LTS backports in security incident process
* Add figure to security incident handling process

### CODE REVIEW

File: security_center/incident_handling_process.rst [hunk 1]
```diff
@@ -4,14 +4,19 @@ Security Incident Handling Process
...
-to the Trusted Firmware security team at security@lists.trustedfirmware.org.
-You may also use the project specific security team aliases
-:doc:`here <mailing_aliases>`. These are private teams of security officers
-who will help verify the security vulnerability, develop and release a fix,
-and disclose the vulnerability details responsibly. The security teams are
-referred to from now on as "us/we/our". We do our best to respond and fix any
-issues as soon as possible.
+If you think you have found a security vulnerability in Trusted Firmware
+infrastructure, then please follow the
+`Linaro security policy <https://www.linaro.org/.well-known/security.txt>`__
```
Comment (sandrine-bailleux-arm) [inline security_center/incident_handling_process.rst:9]: Should we also / rather refer to https://www.linaro.org/vdp?

**Review by sandrine-bailleux-arm — APPROVED** [review]

**Comment by jbech-linaro** [comment]
Same comment as Sandrine, thanks for adding the image!
`Reviewed-by: Joakim Bech <joakim.bech@linaro.org>`

**Review by sandrine-bailleux-arm — APPROVED** [review]

File: security_center/incident_handling_process.rst (hunk 1 above)
Comment (danh-arm) [reply security_center/incident_handling_process.rst:9]: OK. I was concerned about the vdp link changing but using both works!

Comment (jbech-linaro) [reply security_center/incident_handling_process.rst:9]: I don't think we have any plans of changing that and if we would, it's still quick and easy to update the link in the docs.

**Review by jbech-linaro — APPROVED** [review]

**Pull request merged**